Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerSendButton.swift

```diff
@@ -5,19 +5,25 @@
 import StreamChat
 import UIKit
 
+/// Button used for final sending or confirming editing messages in ComposerView.
 public typealias ChatMessageComposerSendButton = _ChatMessageComposerSendButton<NoExtraData>
 
+/// Button used for final sending or confirming editing messages in ComposerView.
 open class _ChatMessageComposerSendButton<ExtraData: ExtraDataTypes>: _ComposerButton<ExtraData> {
```

Comment:
Maybe add some documentation that complete customization can be achieved by subclassing due to `mode` behavior of this component? Or maybe rethink the behavior, I am trying to come up with something...